docs/troubleshooting.md

```diff
@@ -1,20 +1,20 @@
 # Kubernetes Troubleshooting
 
-Get help when you have problems with your Kubernetes setup. This page is divided into three core sections that correspond to the common setup issues that may occur as a result of integrating your Kubernetes cluster with Tanzu Observability.
+Get help when you have problems with your Kubernetes setup. This page is divided into three core sections that correspond to the common setup issues that may occur as a result of integrating your Kubernetes cluster with Operations for Applications.
 
-1. [No data flowing into Tanzu Observability](#no-data-flowing-into-tanzu-observability)
-2. [Missing or incomplete data flowing into Tanzu Observability](#missing-or-incomplete-data-flowing-into-tanzu-observability)
+1. [No data flowing into Operations for Applications](#no-data-flowing-into-operations-for-applications)
+2. [Missing or incomplete data flowing into Operations for Applications](#missing-or-incomplete-data-flowing-into-operations-for-applications)
 3. [Running workloads not being discovered or monitored](#running-workloads-not-being-discovered-or-monitored)
 
 For an in depth overview of the integration and how it is deployed, please refer to our [github page](https://github.com/wavefrontHQ/observability-for-kubernetes).
 
-**Note:** If you are currently using the helm managed and installed version of the wavefront proxy and collector, please refer to our legacy troubleshooting guide for instructions on how to troubleshoot your integration.
+**Note:** If you are currently using the helm managed and installed version of the Wavefront Proxy and Collector, please refer to our legacy troubleshooting guide for instructions on how to troubleshoot your integration.
```

Comment:
**Note:** If you are currently using the Helm-managed and installed version of the Wavefront proxy and Collector, please refer to our legacy troubleshooting guide for instructions on how to troubleshoot your integration.